tests/e2e/sendEmail.test.ts

```diff
@@ -44,13 +49,13 @@ describe('web3mail.sendEmail()', () => {
     });
     // avoid race condition with subgraph indexation
     await sleep(5_000);
-  }, 3 * MAX_EXPECTED_BLOCKTIME + 5_000);
+  }, 2 * MAX_EXPECTED_BLOCKTIME + MAX_EXPECTED_WEB2_SERVICES_TIME + 5_000);
```

Comment:
2 tx => 2 * (1 block to submit tx + 1 block for confirmation) => 4
```suggestion
  }, 4 * MAX_EXPECTED_BLOCKTIME + MAX_EXPECTED_WEB2_SERVICES_TIME + 5_000);
```